source/includes/concept/_overview.md

```diff
@@ -19,6 +19,23 @@ Last but not least, the RW API and its docs are made by humans, who will occasio
 
 This section covers a list of topics you should be familiar with before using the API. The concepts described in this section span across multiple API endpoints and are fundamental for a better understanding of how to interact with the RW API.
 
+### Caching
+
+HTTP caching is a technique that stores a copy of a given resource and serves it back when requested. When a cache has a requested resource in its store (also called a _cache hit_), it intercepts the request and returns its copy instead of re-computing from the originating server. If the request is not yet stored in cache (also called _cache miss_), it is forwarded to the server responsible for handling it, the response is computed and stored in cache to serve future requests. This achieves several goals: it eases the load of the server that doesn’t need to serve all requests itself, and it improves performance by taking less time to transmit the resource back. You can read more about HTTP caching in the [Mozilla Developer Network docs on HTTP caching](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching).
+
+The RW API has a system-wide HTTP cache that you may use to cache your requests. In the context of the RW API, HTTP caching is opt-in, meaning that by default no caching mechanism is applied to any request - if you'd like your endpoints to benefit from caching, you need to explicitly implement it. Please refer to the [developer docs on HTTP caching](/developer.html#http-caching) for more details on how to implement caching for your API resources.
+
+The default cache time to live (TTL) applied to the responses stored in the RW API's cache is 3 days. For the purposes of caching, only responses of GET requests with successful response codes (such as 200, 203, 204 or 302) are considered for caching. Also, for security reasons, authentication, authorization or user related information is never stored in cache.
```

Comment:
The default cache time to live (TTL) applied to the responses stored in the RW API's cache is 3 days, but specific endpoints may specify a different expiration time for their responses.